packages/browser/src/plugins/destinations/google-tag-manager.ts

```diff
@@ -34,42 +30,25 @@ type GoogleTagManagerSettings = {
  * https://github.com/segmentio/analytics.js-integrations/blob/master/integrations/google-tag-manager/lib/index.js
  */
 const googleTagManager: DestinationFactory<GoogleTagManagerSettings> = ({
-  measurementId = [],
   trackAllPages = false,
   trackNamedPages = true,
   trackCategorizedPages = true,
 }) => {
-  const measurementIds = (
-    Array.isArray(measurementId) ? measurementId : [measurementId]
-  ).filter(Boolean)
-
-  const baseEvent = ({ event }: Context) => {
-    return {
+  const pushEvent = (event: HightouchEvent) => {
+    window.dataLayer.push({
       ...(event.userId && {
-        user_id: event.userId,
+        userId: event.userId,
```

Comment:
I think we should snake case `user_id` here like we do for `gtag` since GA4 expects that format for user IDs.

https://developers.google.com/analytics/devguides/collection/ga4/user-id?client_type=gtm#send_user_ids